src/item.cpp

```diff
@@ -2493,16 +2493,11 @@ int item::bash_resist() const
         return resist;
     }
     if (item::item_tags.count("leather_padded") > 0){
-        l_padding = volume() * (float(get_coverage()) / 100) / 2.5;
-        if (l_padding > 5 ){
-            l_padding = l_padding / 2.5;   //Hard cap so coats don't become solid steel
-        }
+               //Max value           Steepness of graph         Center of 'S'
```

Comment:
That is a much better way to do that. Quick style question, is it better to have the variables get declared within each of the if leather_padded and kevlar_padded statements or just once at the beginning of the whole bash_resist and cut_resist functions?